codespell_lib/data/dictionary_code.txt

```diff
@@ -13,6 +13,7 @@ dont->don't
 dur->due
 endcode->encode
 errorstring->error string
+ether->either
```

Comment:
-1 on this one, `ether` is a chemistry term AFAIK (i.e., used outside of code / just rare)

Reply:
Okay I've dropped it entirely now. It was in rare but the first 7 pages of grep.app at least are genuine code usages, so I don't think it's actually very rare really.